dotnetv3/MediaConvert/README.md

```diff
@@ -38,9 +38,9 @@ For prerequisites, see the [README](../README.md#Prerequisites) in the `dotnetv3
 
 Code excerpts that show you how to call individual service functions.
 
-- [CreateJob](Scenarios/CreateJob/CreateJob.cs#L24)
-- [GetJob](Scenarios/CreateJob/CreateJob.cs#L24)
-- [ListJobs](Scenarios/CreateJob/CreateJob.cs#L24)
+- [CreateJob](Scenarios/CreateJob/CreateJob.cs#L23)
+- [GetJob](Scenarios/CreateJob/CreateJob.cs#L23)
```

Comment:
Are all of these supposed to point to CreateJob.cs on the same line?

Reply:
Yes, it is ok, that is where the setup (the first snippet of all of the examples) is.